### PR TITLE
pubsub: make Publisher log less chatty

### LINKED ISSUE #1681: Publisher.publish - Reducing log level for individual publishes
There seems to be excessive number of INFO level log messages when publishing messages.  Revisit and reduce the log levels where appropriate.

log

### PR DESCRIPTION
**Pull request by pongad**
Fixes #1681.

### CODE REVIEW

File: google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java [hunk 1]
```diff
@@ -236,7 +236,7 @@ public TopicName getTopicName() {
       if (!messagesBundle.isEmpty()) {
         setupDurationBasedPublishAlarm();
       } else if (currentAlarmFuture != null) {
-        logger.log(Level.INFO, "Cancelling alarm");
+        logger.log(Level.FINER, "Cancelling alarm, no more messages");
```
Comment (garrettjonesgoogle) [inline google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java]: I think some of the logs here could be `FINE` instead of `FINER`.

Comment (lesv) [inline google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java]: FINER - works here.

File: google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java [hunk 2]
```diff
@@ -248,7 +248,7 @@ public TopicName getTopicName() {
     messagesWaiter.incrementPendingMessages(1);
 
     if (bundleToSend != null) {
-      logger.log(Level.INFO, "Scheduling a bundle for immediate sending.");
+      logger.log(Level.FINER, "Scheduling a bundle for immediate sending.");
```
Comment (lesv) [inline google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java]: FINER works here

File: google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java [hunk 3]
```diff
@@ -263,7 +263,7 @@ public void run() {
     // be sent in its own bundle immediately.
     if (hasBundlingBytes() && messageSize >= getMaxBundleBytes()) {
       logger.log(
-          Level.INFO, "Message exceeds the max bundle bytes, scheduling it for immediate send.");
+          Level.FINER, "Message exceeds the max bundle bytes, scheduling it for immediate send.");
```
Comment (lesv) [inline google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java]: FINER works here

File: google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java [hunk 4]
```diff
@@ -280,13 +280,13 @@ public void run() {
...
       long delayThresholdMs = getBundlingSettings().getDelayThreshold().getMillis();
-      logger.log(Level.INFO, "Setting up alarm for the next {0} ms.", delayThresholdMs);
+      logger.log(Level.FINER, "Setting up alarm for the next {0} ms.", delayThresholdMs);
       currentAlarmFuture =
           executor.schedule(
               new Runnable() {
                 @Override
                 public void run() {
-                  logger.log(Level.INFO, "Sending messages based on schedule.");
+                  logger.log(Level.FINER, "Sending messages based on schedule.");
```
Comment (lesv) [inline google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java]: FINER seems OK here.

File: google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java [hunk 5]
```diff
@@ -280,13 +280,13 @@ public void run() {
   private void setupDurationBasedPublishAlarm() {
     if (!activeAlarm.getAndSet(true)) {
       long delayThresholdMs = getBundlingSettings().getDelayThreshold().getMillis();
-      logger.log(Level.INFO, "Setting up alarm for the next {0} ms.", delayThresholdMs);
+      logger.log(Level.FINER, "Setting up alarm for the next {0} ms.", delayThresholdMs);
```
Comment (lesv) [inline google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java]: FINER seems to work here.

**Comment by garrettjonesgoogle** [comment]
LGTM, since @lesv thinks `FINER` works fine for the logging level.

**Pull request merged**